radio/src/CMakeLists.txt

```diff
@@ -18,6 +18,7 @@ option(AUTOSOURCE "Automatic source detection in menus" ON)
 option(AUTOSWITCH "Automatic switch detection in menus" ON)
 option(JITTER_MEASURE "Enable ADC jitter measurement" OFF)
 option(JITTER_FILTER "Enable ADC jitter filtering" ON)
+option(WATCHDOG_DISABLED "Disable hardware Watchdog on Horus" ON)
```

Comment:
Is this option *really* needed? If yes it should be only in the PCB=HORUS part, no?